app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
```

Comment:
Tìm hiểu cho chị `I18n in rails`
=> Sau đó, dùng i18n cho toàn bộ text trong toàn bộ file html nhé